### ISSUE TITLE
Task: Documenter l'accès aux services de Nplex

### ISSUE THREAD

**Issue by fradrwe5** [CUPUM/nplex#292]
### Task description

Valider et documenter l'accès aux différents services de Nplex.

### List of actions

- [ ] Action : Documenter l'accès aux différents services (Neon, Vercel, AWS) (service, compte, type d’accès, statut accès actuel, identifiant, utilisation)
- [ ] Action :  Identifier une stratégie pour stocker les variables d’environnement

**Comment by fradrwe5** [comment]
| Service | Utilisation | Compte et accès | Statut d'accès actuel | Lien |
|---------|-------------|-----------------|---------------------|------|
| Neon    | Hébergement de la base de données | Le nouveau gestionnaire doit se créer un compte sur Neon (associé à une adresse courriel ou un compte GitHub). Emmanuel ou Fred peuvent ensuite donner accès à la base de données de production/développement. | Projet Neon géré par Emmanuel, Fred a accès (peut ajouter des gens) | [Neon](https://console.neon.tech/app/projects/weathered-voice-90200607?branchId=br-gentle-poetry-85803328&database=prod) |
| Vercel  | Hébergement applicatif | Le nouveau gestionnaire doit se créer un compte sur Vercel. Emmanuel peut ensuite transférer le projet au gestionnaire. | Emmanuel a accès | [Vercel](https://vercel.com/) |
| AWS     | S3 et service de traitement d’image | Le nouveau gestionnaire doit se créer un compte sur AWS. Emmanuel peut ensuite ajouter le gestionnaire au projet AWS. | Emmanuel a accès | [AWS](https://signin.aws.amazon.com/signup?request_type=register) |

**Comment by fradrwe5** [comment]
Salut @sjinko, ci-dessus la liste des comptes et accès aux services tels que décrits par Emmanuel dans son courriel. Est-ce que c'est ça qui était attendu?

**Comment by sjinko** [comment]
Oui, est-ce que ca comprend les comptes accessibles avec les fichiers env ?
